### PR TITLE
Bump lts to 17.2

### PR DESCRIPTION
**Pull request by sashasashasasha151**
#### Related changes (conditional)

- [ ] Tests
  - If I added new functionality, I added tests covering it.
  - If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- [ ] Documentation
  - I checked whether I should update the docs and did so if necessary:
    - [README](/README.md);
    - Haddock;
    - [Examples](/examples/).

- [ ] Public contracts
  - Any modifications of public contracts comply with the [Evolution
  of Public Contracts](https://www.notion.so/serokell/Evolution-of-Public-Contracts-2a3bf7971abe4806a24f63c84e7076c5) policy.
  - I added an entry to the [changelog](../tree/master/CHANGES.md) if my changes are visible to the users
        and
  - provided a migration guide for breaking changes if possible.

#### Stylistic guide (mandatory)

- [ ] Style
  - My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
  - My code complies with the [style guide](../tree/master/docs/code-style.md).
  - Each commit of this pull request contains a description.
    - For new features, this description contains the use case for the new functionality.
    - For bug fixes, it describes both the attacked problem and a solution for it.
    - For dependency version changes description shortly enlists features and breaking changes of the new library version or contains a link to its changelog.

### CODE REVIEW

File: servant-util/src/Servant/Util/Combinators/Pagination.hs [hunk 1]
```diff
@@ -13,10 +13,12 @@ import Control.Lens ((<>~), (?~))
 import Data.Default (Default (..))
 import qualified Data.Swagger as S
 import qualified Data.Text as T
-import Servant ((:>), HasServer (..), QueryParam)
+import Servant ((:>), DefaultErrorFormatters, ErrorFormatters, HasContextEntry, HasServer (..),
+                QueryParam)
```
Comment (Martoon-00) [inline servant-util/src/Servant/Util/Combinators/Pagination.hs:17]: Btw, please enable `stylish-haskell` on save and reformat all the imports in the modules you update in this PR. I get extra diff when trying to update this module somehow.

Comment (gromakovsky) [reply servant-util/src/Servant/Util/Combinators/Pagination.hs:17]: AFAIK Sasha uses version 0.11, that may be the reason.

Comment (Martoon-00) [reply servant-util/src/Servant/Util/Combinators/Pagination.hs:17]: Oh really, that's a nice meme indeed :facepalm: 

I use 0.12.2.0.

I thought only ormolu changes its behaviour depending on version, not on config, so :fuuu: if stylish-haskell also suffers from this.

**Review by Martoon-00 — APPROVED** [review]

File: servant-util/src/Servant/Util/Combinators/Pagination.hs (hunk 1 above)
Comment (gromakovsky) [reply servant-util/src/Servant/Util/Combinators/Pagination.hs:17]: In 0.12 they [switched](https://github.com/jaspervdj/stylish-haskell/blob/main/CHANGELOG) to `ghc-lib-parser` from `haskell-src-exts`, so I think such minor inconsistencies are somewhat expected and justified. That sounds like a very crucial change, but a good one.

**Pull request merged**